dts/bindings/adc/adi,ad4130.yaml

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) 2025 Analog Devices, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Bindings for the ADI AD4130 Analog-to-Digital Converter.
```

Comment:
Please extend the description with a devicetree example, and a link to the devicetree bindings header

Reply:
I have updated. Could you take a look?